test/libsolidity/SMTChecker.cpp

```diff
@@ -751,6 +751,24 @@ BOOST_AUTO_TEST_CASE(division_truncates_correctly)
 	)";
 	CHECK_SUCCESS_NO_WARNINGS(text);
 }
+BOOST_AUTO_TEST_CASE(regression_test)
```

Comment:
I don't think regression test is a good name for this.

Reply:
any suggestions?

Reply:
I'd suggest `unsupported_types`